cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/UserTasksParameters.java

```diff
@@ -29,10 +34,30 @@ public UserTasksParameters(HttpServletRequest request) {
   @Override
   protected void initParameters() throws UnsupportedEncodingException {
     super.initParameters();
-    _userTaskIds = ParameterUtils.userTaskIds(_request);
+    _taskIds = ParameterUtils.taskIds(_request);
```

Comment:
These names are good!
Please see my previous comment on `taskIds`, and please ensure the same name change and make them consistent for getters, parsers, field names, etc.